src/Beta.js

```diff
@@ -9,5 +9,13 @@ const BetaWarning = function ( props ) {
   );
 };  // End BetaWarning(<>)
 
+const About = function ( props ) {
+  return (
+    <Menu.Item>
+      <a href='https://github.com/codeforboston/cliff-effects#cliff-effects'>About</a>
```

Comment:
You can add `target="_blank"` to the `<a>` tag to open in it in a new window. Not a dealbreaker, but worth nothing.

Reply:
Good point! Though this may be moot now. Still, we should keep this in mind for the future since we have the 'Feedback' button to consider.

Reply:
As per discussion on [Issue 136](https://github.com/codeforboston/cliff-effects/issues/136) create blank About Page instead of pointing to repo README.  Content will be populated later.  Since About will not point to an external site, we don't need to open in new tab (in other cases, would agree!)